modules/products/jira/variables.tf

```diff
@@ -114,6 +113,12 @@ variable "shared_home_snapshot_id" {
   }
 }
 
+variable "image_repository" {
+  description = "Jira image repository"
+  type        = string
+  default     = "atlassian/jira-software"
```

Comment:
We should have validation for this as the valid values are restricted to a set.

Reply:
I don't want to include validation for the set as this enables testing different images as well. I was even thinking about including this variable for other products so people can test with customized images but I don't want to expand the scope.

We have documentation for official values so users should be able to know what can be plugged here.